tdrs-backend/reports/zap.conf

```diff
@@ -0,0 +1,2 @@
+10036	IGNORE	(Server leaks version information via Server HTTP Response Header field)
```

Comment:
adding an in-line comment: in my note for this one, I have that this finding is a false positive because the server information is not a secret for our open source project.